INSTALL.md

```diff
@@ -60,24 +60,46 @@ You will need:
  * PhysFS
  * oggenc (to generate music files)
 
-On Ubuntu (and probably any other Debian-based system), you can use the following command to install all required packages:
+On Ubuntu (and probably any other Debian-based system) you can use the following commands.
+
+The apt-get update is optional but recommended to use before install packages:
+
+```
+  $ sudo apt-get update
+```
+
+Use the following command to install all required packages:
 ```
-  $ apt-get install build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
+  $ sudo apt-get install --no-install-recommends build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
 ```
+key `--no-install-recommends` is optional.
 
 Make sure you install the packages along with header files (often distributed in separate *-dev packages). If you miss any requirements,
 CMake should warn you.
 
-To compile colobot, run your favorite shell and download colobot source files:
+Create a folder to sources. And enter into it:
+
 ```
-  $ git clone https://github.com/colobot/colobot.git /path/to/colobot/sources
-  $ cd /path/to/colobot/sources
-  $ git submodule update --init
+  mkdir colobot_sources
+  cd colobot_sources
+```
+
+Download colobot source files from github:
+```
+  $ git clone https://github.com/colobot/colobot.git
 ```
-If you want to compile development branch, change the first command to:
+If you want to compile development branch, change command to:
 ```
   $ git clone -b dev https://github.com/colobot/colobot.git /path/to/colobot/sources
 ```
+
+Also we need media resources for compile. Enter to colobot folder and download it:
```

Comment:
I'm not expert at English, but I think there is room for improvement. But I might be wrong there.
I think first sentence should end with `resources for compilation`. 
There is also something wrong with second sentence. I think it should be `Enter into colobot folder`, also not sure if it should be `it` or `them` at the end of sentence.

Reply:
Im not expert in English too. A likely solution:

    Media resources also is needed for build. Enter to \colobot folder and download it too: